source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+     - 4.38 hours
+     - 21.92 minutes
+     - 5.04 minutes
+     - 43.20 seconds
+   * - 99.99% ("four nines")
+     - 52.60 minutes
+     - 4.38 minutes
+     - 1.01 minutes
+     - 8.64 seconds
+   * - 99.995% ("four nines five")
+     - 26.30 minutes
+     - 2.19 minutes
+     - 30.24 seconds
+     - 4.32 seconds
+   * - 99.999% ("five nines")
+     - 5.26 minutes
+     - 26.30 seconds
+     - 6.05 seconds
+     - 864.00 milliseconds
+  
+How is high availability achieved?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are 3 key components to achieve high availability:
+
+1. **Infrastructure** - This is the physical or virtual hardware that database systems rely on to run. Without enough infrastructure (VM’s, networking, etc.) there cannot be high availability. The easiest example is: ``there is no way to make a single server highly available``.
+2. **Topology management** - This is the software management related specifically to the database and managing its ability to stay consistent in the event of a failure. Many clustering or synchronous replication solutions offer this capability out of the box. However, for asynchronous replication this is handled by additional software. 
+3. **Connection management** - This is the software management related specifically to the networking and connectivity aspect of the database. Clustering solutions typically bundle with a connection manager. However, in asynchronous clusters deploying a connection manager is mandatory for high availability.
+
+This solution is based on a tightly coupled database cluster. It offers a high availability level of 99.995% when coupled with the Group Replication setting ``group_replication_consistency=AFTER``.
+
+.. image:: _static/images/group-replication-1.png
+
+
+
+.. _failovers:
+
+Failovers
+--------------------
+
+A database failure or configuration change that requires a restart should not affect the stability of the database infrastructure, if it is properly planned and architected. Failovers are an integral part of a stability strategy and aligning the business requirements for availability and uptime with failover methodologies is critical. 
+
+The following are the 3 main types of failovers that can occur in database environments:
+
+* **Planned failover**. A planned failover is a failover that has been scheduled in advance or occurs at a regular interval. There can be many reasons for planned failovers including patching, large data operations, retiring existing infrastructure, or simply to test the failover strategy.
+
+* **Unplanned failover**. An unplanned failover is what occurs when a database has unexpectedly become unresponsive or experiences instability. This could also include emergency changes that do not fall under the planned failover cadence or scheduling parameters. Unplanned failovers are generally considered higher risk operations due to the high stress and high potential for either data corruption or data fragmentation.
```

Comment:
This could
Unclear what "This" is referring to.

high potential for data corruption or data fragmentation
(the either is redundant)

Reply:
fixed